src/components/books/Book.jsx

```diff
@@ -1,18 +1,21 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { useDispatch } from 'react-redux';
+import { buildStyles, CircularProgressbar } from 'react-circular-progressbar';
+import 'react-circular-progressbar/dist/styles.css';
 
 import styles from './Book.module.scss';
 import { deleteBook } from '../../redux/books/books';
 
 const Book = ({ book }) => {
-  const {
-    item_id: itemID, author, title, category, completed,
-  } = book;
+  const { item_id: itemID, author, title, category, completed } = book;
```

Comment:
- Please, kindly update these lines of code to the way it was before as this is throwing a linter error and preventing the app to run to fix this issue:

![Screenshot from 2023-01-28 13-26-53](https://user-images.githubusercontent.com/59397159/215266649-cf776112-816d-45e4-b587-0a4e60864e0b.png)

Reply:
### FORM TITLE AND HEADER

- Please, kindly use the correct color for the form title and the header of your app to match the design as much as you can below:
- Use the correct font family as yours is very obvious that it's different compared to the design.


![Screenshot from 2023-01-28 13-37-38](https://user-images.githubusercontent.com/59397159/215266917-031a4b7b-e5a9-465e-9f1a-dd76b34e8e67.png)
![Screenshot from 2023-01-28 13-36-42](https://user-images.githubusercontent.com/59397159/215266919-ca86e150-54fc-4bc7-a9e3-57afe58268ed.png)